styles/components/header-nav.scss

```diff
@@ -0,0 +1,140 @@
+@import "../variables.scss";
+
+.header-nav {
+  color: var(--colour-neutral-000);
+  background: linear-gradient(180deg, #141515 0%, rgba(20, 21, 21, 0) 100%);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: fixed;
+  width: 100%;
+  top: 0;
+  z-index: 10;
+
```

Comment:
The navbar isn't centred on larger screens.

![01](https://user-images.githubusercontent.com/47471007/233158141-8cad7bcf-a177-4cee-b087-423b3020eb5d.png)

I think adding this should fix it.
```
left: 0;
right: 0;
 ```


Other than that, looks good!

Reply:
Thanks for catching that, Marcus! I see another styling issue for the footer on large screens as well. I should not forget test on large screens from now on 😅